src/Kunstmaan/NodeBundle/Helper/NodeAdmin/NodeAdminPublisher.php

```diff
@@ -263,4 +277,50 @@ public function createPublicVersion(
 
         return $newNodeVersion;
     }
+
+    /**
+     * @param Request         $request
+     * @param NodeTranslation $nodeTranslation
+     */
+    public function chooseHowToPublish(Request $request, NodeTranslation $nodeTranslation)
+    {
+        if ($request->request->has('publish_later') && $request->get('pub_date')) {
+            $date = new \DateTime(
+                $request->get('pub_date') . ' ' . $request->get('pub_time')
+            );
+            $this->publishLater($nodeTranslation, $date);
+            $this->session->getFlashBag()->add(
+                FlashTypes::SUCCESS,
+                $this->translator->trans('kuma_node.admin.publish.flash.success_scheduled')
```

Comment:
I'll have a look into this, because it means this can be done without a BC break then.
But, I think this is a bit cleaner, so'll make a different PR to go into the 6 branch.